doc/rst/tools/mason/guide/basicusage.rst

```diff
@@ -14,8 +14,7 @@ To initialize a new mason package, run ``mason new``. The same can also be done
     cd newPackage
     mason init
 
-
-This starts an interactive session which walks a user through the process of creating a project using Mason. This is highly recommended for new users.
+This setups the folder to have the structure of a mason package. Particularly, it creates a `Mason.toml` file and the `src` folder with the package main module.
```

Comment:
"setups the", I think you mean "sets up a"? So, "This sets up a folder to have..."

Reply:
yes! thanks fixed